.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Bug Report
+about: Create a bug report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+------------------------------- delete below -------------------------------
+
+INSTRUCTIONS
+==========
+
+If you do not follow the below instructions, your issue may be dismissed.
```

Comment:
Maybe reword this a little, we want to be nice to our customers

Reply:
Something like: we may be forced to close your issues to keep our issue quality high.